freecad/gridfinity_workbench/feature_construction.py

```diff
@@ -529,7 +529,7 @@ def make(self, obj: fc.DocumentObject) -> Part.Shape:
             fc.Console.PrintMessage(
                 "scooop could not be made due to bin selected parameters\n",
             )
-            return None
+            return None  # type: ignore[return-value]
```

Comment:
We cannot ignore this nor return None here.

We should or, make the return value accept None, which does not have my preference.

Or, raise an error which is caught by the calling code. The calling code will be responsible if this error is going to be blocking or just a warning.